src/familiar.ts

```diff
@@ -89,5 +89,11 @@ export function freeFightFamiliar(): Familiar {
     familiarValue.push([$familiar`Stocking Mimic`, mimicValue]);
   }
 
+  for (const familiar of $familiars`Robortender, Hobo Monkey, Cat Burglar, Leprechaun`) {
```

Comment:
Robortender should be a higher value than 1 since it drops candy and junk - maybe 200 is a good estimate?

Reply:
Not yet addressed- I feel like I'd want to address this as a separate familiar valuing PR, this is definitely just a bandage to make sure that something sensible comes up. My temptation is to make this a new issue that I can work on at a later date?